### ISSUE TITLE
Intrusive tabindex set on panzoom owner element

### ISSUE THREAD

**Issue by mnogueron** [anvaka/panzoom#80]
Whenever creating a new panzoom component, the owner element gets a quite intrusive `tabindex` attribute due to this line:

https://github.com/anvaka/panzoom/blob/edc1fa82d6ba60c93cf7bbcc20bd90e4edf13588/lib/domController.js#L17

If you have this kind of DOM tree:
```
<div id="container">
   <div id="otherElements">...</div>
   <div id="panzoom"></div>
</div>
```
`container` will get the `tabindex` attribute and will be focused on click. On Chrome a blue outline on the whole `container` will be shown, which might be wrong in most cases.

It would be great to have a way to disable this behavior and let devs define `tabindex` wherever they want.

**Comment by anvaka** [comment]
The primary reason for this is to let keyboard shortcuts work. The blue outline can be removed `:focus` selector.

I'm thinking maybe we should add a new option to the panzoom, that would disable `tabindex`? How would something like this work for you:

``` js
panzoom(el, {
  setTabIndex: false
});
```

?

**Comment by mnogueron** [comment]
Thank you @anvaka for your answer. I realised afterward that without `tabindex` no focus for keyboard event.

Having the option to disable the `tabindex` would be awesome, but keyboard shortcuts would not work anymore. Maybe something like would be more explanatory and would not hide the side effect of removing `tabindex`.
```
panzoom(el, {
   disableKeyboardInteraction: true
});
```

**Comment by bakuzan** [comment]
I'm all for the "disableKeyboardInteraction" options.

I'd also like to chime in on a related note.
Even if the disabling feature isn't implemented, the tabindex should be changed to '0' so as to not break the nature tab order.

From [MDN tabindex article](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex):
> It is not recommended to give positive values to your elements. You will end up jumping between them, and it will be confusing to manipulate between your element's tabindex attribute values. Instead, focus on writing them in a suitable DOM sequence.

**Comment by anvaka** [comment]
Thank you for the suggestion and implementation! This should be available starting from `7.2.0`.
